### PR TITLE
fix: [titlebar] Adjust button size

### PR DESCRIPTION
**Pull request by GongHeng2017**
According to the design requirements, adjust the button size.

Log: fix issue
Bug: https://pms.uniontech.com/bug-view-197585.html

### CODE REVIEW

**Review by pppanghu77 — DISMISSED** [review]

File: src/plugins/filemanager/core/dfmplugin-titlebar/views/navwidget.cpp [hunk 1]
```diff
@@ -184,4 +192,24 @@ void NavWidget::initConnect()
     connect(DevProxyMng, &DeviceProxyManager::protocolDevUnmounted, this, &NavWidget::onDevUnmounted);
     connect(DevProxyMng, &DeviceProxyManager::blockDevRemoved, this, &NavWidget::onDevUnmounted);
     connect(DevProxyMng, &DeviceProxyManager::protocolDevRemoved, this, &NavWidget::onDevUnmounted);
+
+#ifdef DTKWIDGET_CLASS_DSizeMode
+    connect(DGuiApplicationHelper::instance(), &DGuiApplicationHelper::sizeModeChanged, this, [this]() {
+        initUiForSizeMode();
```
Comment (itsXuSt) [inline src/plugins/filemanager/core/dfmplugin-titlebar/views/navwidget.cpp]: 这里没必要装 lambda 里头啊，把它声明成一个槽函数直接写函数指针就可以了撒

**Review by itsXuSt — CHANGES_REQUESTED** [review]

File: src/plugins/filemanager/core/dfmplugin-titlebar/views/navwidget.h [hunk 2]
```diff
@@ -44,6 +44,7 @@ public slots:
 private:
     void initializeUi();
     void initConnect();
+    void initUiForSizeMode();
```
Comment (Clauszy) [inline src/plugins/filemanager/core/dfmplugin-titlebar/views/navwidget.h]: changeSizeMode

**Review by Clauszy — CHANGES_REQUESTED** [review]

File: src/plugins/filemanager/core/dfmplugin-titlebar/views/navwidget.cpp (hunk 1 above)
Comment (GongHeng2017) [reply src/plugins/filemanager/core/dfmplugin-titlebar/views/navwidget.cpp]: done

File: src/plugins/filemanager/core/dfmplugin-titlebar/views/navwidget.h (hunk 2 above)
Comment (GongHeng2017) [reply src/plugins/filemanager/core/dfmplugin-titlebar/views/navwidget.h]: done

**Review by Clauszy — COMMENTED** [review]
+1

**Review by Johnson-zs — APPROVED** [review]

**Review by itsXuSt — APPROVED** [review]

**Review by Clauszy — APPROVED** [review]

**Pull request merged**